utils/helpers.js

```diff
@@ -1,6 +1,9 @@
 export const range = (length, start = 0, step = 1) =>
   Array.from(Array(length), (_, i) => start + i * step)
 
+export const formatNumber = (number) =>
+  new Intl.NumberFormat('en-GB', { maximumSignificantDigits: 3 }).format(number)
```

Comment:
I am really happy you used `Intl.NumberFormat`, this is the most appropriate way.

I tested it, and I am not sure `maximumSignificantDigits` does exactly what you want. Just test it:

```js
new Intl.NumberFormat('en-GB', { maximumSignificantDigits: 3 }).format(123456.789) // outputs 123,000
```

I propose to make the function more configurable:

```suggestion
export const formatNumber = (number, { locale = 'en-GB', ...options } = {}) =>
  new Intl.NumberFormat(locale, options).format(number)
```